### PR TITLE
TestCases: PP4_553_557_558_559

### PR DESCRIPTION
**Pull request by QuocLe7**
- TestCases: PP4_553_557_558_559
- Platform: Android
- Apk version: 6.4.0
- Date: 24/08/2018

### CODE REVIEW

File: tests/android/placement/golden_templates/local_ads_request_expected_content.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# CONTAINS - means [your expected value] contains in [actual data]
+#
+####NUMBER COMPARISON####
+# EQUAL = mean [your expected number] = [actual number]
+# LESS_THAN = mean [your expected number] < [actual number]
+# LESS_THAN_OR_EQUAL = mean [your expected number] <= [actual number]
+# GREATER_THAN = mean [your expected number] > [actual number]
+# GREATER_THAN_OR_EQUAL = mean [your expected number] >= [actual number]
+
+make="ANYTHING"
```
Comment (anhtaicit) [inline tests/android/placement/golden_templates/local_ads_request_expected_content.py]: Request Json should contain following field:- 
 "request info", "App info", 'device info" and 'system info",  "placements"

File: tests/android/placement/golden_templates/local_ads_response_expected_content.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+####STRING COMPARISON####
+# CONTAINS - means [your expected value] contains in [actual data]
+#
+####NUMBER COMPARISON####
+# EQUAL = mean [your expected number] = [actual number]
+# LESS_THAN = mean [your expected number] < [actual number]
+# LESS_THAN_OR_EQUAL = mean [your expected number] <= [actual number]
+# GREATER_THAN = mean [your expected number] > [actual number]
+# GREATER_THAN_OR_EQUAL = mean [your expected number] >= [actual number]
+
```
Comment (anhtaicit) [inline tests/android/placement/golden_templates/local_ads_response_expected_content.py]: Response Json file should contain the following field for valid placement id:-"placement_reference_id", "ad_markup"
[id:,adType,campaign,callToActionUrl,countdown,showClose,cta_overlay,expiry ,app_id,showCloseIncentivized,delay,preBundle:,postBundle:,callToActionDest:,tpat:url:,videoWidth: ,videoHeight: ,md5:,size: ,chk:,retryCount: ,asyncThreshold: ,ad_token:,
video_object_id:,templateURL:,templateId:,templateFormat:,templateSettings:,info:,sleep]

File: tests/android/placement/golden_templates/local_config_request_expected_content.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+####STRING COMPARISON####
+# CONTAINS - means [your expected value] contains in [actual data]
+#
+####NUMBER COMPARISON####
+# EQUAL = mean [your expected number] = [actual number]
+# LESS_THAN = mean [your expected number] < [actual number]
+# LESS_THAN_OR_EQUAL = mean [your expected number] <= [actual number]
+# GREATER_THAN = mean [your expected number] > [actual number]
+# GREATER_THAN_OR_EQUAL = mean [your expected number] >= [actual number]
+
```
Comment (anhtaicit) [inline tests/android/placement/golden_templates/local_config_request_expected_content.py]: Request should contain placement ID

File: tests/android/placement/golden_templates/local_report_ad_request_expected_content.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# CONTAINS - means [your expected value] contains in [actual data]
+#
+####NUMBER COMPARISON####
+# EQUAL = mean [your expected number] = [actual number]
+# LESS_THAN = mean [your expected number] < [actual number]
+# LESS_THAN_OR_EQUAL = mean [your expected number] <= [actual number]
+# GREATER_THAN = mean [your expected number] > [actual number]
+# GREATER_THAN_OR_EQUAL = mean [your expected number] >= [actual number]
+
+make="ANYTHING"
```
Comment (anhtaicit) [inline tests/android/placement/golden_templates/local_report_ad_request_expected_content.py]: Response Json file should contain the following field:- 
"api_endpoints","placement_info","config","will_play_ad","playback"

File: tests/android/placement/golden_templates/local_report_ad_response_expected_content.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+####STRING COMPARISON####
+# CONTAINS - means [your expected value] contains in [actual data]
+#
+####NUMBER COMPARISON####
+# EQUAL = mean [your expected number] = [actual number]
+# LESS_THAN = mean [your expected number] < [actual number]
+# LESS_THAN_OR_EQUAL = mean [your expected number] <= [actual number]
+# GREATER_THAN = mean [your expected number] > [actual number]
+# GREATER_THAN_OR_EQUAL = mean [your expected number] >= [actual number]
+
```
Comment (anhtaicit) [inline tests/android/placement/golden_templates/local_report_ad_response_expected_content.py]: No need to verify /reportAd

**Comment by QuocLe7** [comment]
I fixed al issues according to you guideline, please refer to https://github.com/anhtaicit/draft_automation/pull/9 PR, thanks.

**Pull request closed without merging**